### PR TITLE
Refs #267: Make react-image-crop SSR friendly

### PR DESCRIPTION
**Pull request by ArtBIT**

### CODE REVIEW

File: lib/ReactCrop.js [hunk 1]
```diff
@@ -221,30 +221,32 @@ function containCrop(prevCrop, crop, imageWidth, imageHeight) {
 }
 
 class ReactCrop extends PureComponent {
-  window = window
+  window = null
 
-  document = document
+  document = null
```
Comment (DominicTobias) [inline lib/ReactCrop.js]: Changes to the lib are not needed

File: webpack.config.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ function getConfig(env) {
       library: 'ReactCrop',
       libraryTarget: 'umd',
       filename: env === 'production' ? 'ReactCrop.min.js' : 'ReactCrop.js',
+      globalObject: 'typeof self !== \'undefined\' ? self : this',
```
Comment (DominicTobias) [inline webpack.config.js]: Can you explain what this is doing?

Comment (ArtBIT) [reply webpack.config.js]: It tells webpack not to use the `window` object explicitly, but to pass `this` instead. https://webpack.js.org/configuration/output/#outputglobalobject

**Comment by ArtBIT** [comment]
Cheers! :+1:

**Pull request merged**